Doc/library/datetime.rst

```diff
@@ -1486,9 +1486,9 @@ Instance methods:
    returned by :func:`time.time`.
 
    Naive :class:`.datetime` instances are assumed to represent local
-   time and this method relies on the platform C :c:func:`mktime`
+   time and this method relies on the platform C :c:func:`locatime_s`
```

Comment:
This is not letting me make suggested modifications here, but looking at the code, sometimes it uses `localtime_s` an sometimes it uses `localtime_r`, but overall I think it's probably too muh detail for these docs anyway. I would instead write this as:

```
Naive :class:`.datetime` instances are assumed to represent local
time and this method relies on platform C functions to perform
the conversion. Since :class:`.datetime` supports a wider range of
values than the platform C functions on many platforms, this
method may raise :exc:`OverflowError` or :exc:`OSError` for times
far in the past or far in the future.
```

Thanks!